app/models/studio.rb

```diff
@@ -1,3 +1,9 @@
 class Studio < ApplicationRecord
   has_many :movies
+
+  def actors_featured
+    movie_ids = movies.pluck(:id)
+    actors_featured = Actor.joins(:movies,:movie_actors).where(movies: {id: movie_ids}).distinct
```

Comment:
This works, but we can make it simpler by adding an additional association: `has_many :actors, through: :movies`. 
The reason for this is that a Studio already has many movies, and a movie has many actors.

Reply:
If we added that, this query becomes

```
actors.distinct
```